[2022-08-13 02:18:26,849] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-12T00:00:00+00:00 [queued]>
[2022-08-13 02:18:26,859] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: csv_dag.final_dag scheduled__2022-08-12T00:00:00+00:00 [queued]>
[2022-08-13 02:18:26,860] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 02:18:26,860] {taskinstance.py:1377} INFO - Starting attempt 1 of 2
[2022-08-13 02:18:26,861] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-13 02:18:26,877] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): final_dag> on 2022-08-12 00:00:00+00:00
[2022-08-13 02:18:26,885] {standard_task_runner.py:52} INFO - Started process 3237 to run task
[2022-08-13 02:18:26,889] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'csv_dag', 'final_dag', 'scheduled__2022-08-12T00:00:00+00:00', '--job-id', '177', '--raw', '--subdir', 'DAGS_FOLDER/testDag.py', '--cfg-path', '/tmp/tmpozzxpfjg', '--error-file', '/tmp/tmplu80ggu4']
[2022-08-13 02:18:26,890] {standard_task_runner.py:80} INFO - Job 177: Subtask final_dag
[2022-08-13 02:18:26,957] {task_command.py:371} INFO - Running <TaskInstance: csv_dag.final_dag scheduled__2022-08-12T00:00:00+00:00 [running]> on host 36e1df4dd6ce
[2022-08-13 02:18:27,036] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=csv_dag
AIRFLOW_CTX_TASK_ID=final_dag
AIRFLOW_CTX_EXECUTION_DATE=2022-08-12T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-12T00:00:00+00:00
[2022-08-13 02:18:27,142] {python.py:173} INFO - Done. Returned value was: None
[2022-08-13 02:18:27,156] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=csv_dag, task_id=final_dag, execution_date=20220812T000000, start_date=20220813T021826, end_date=20220813T021827
[2022-08-13 02:18:27,222] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-08-13 02:18:27,253] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
